app/(marketing)/page.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, buttonVariants } from "@/components/ui/button";
+import Link from "next/link";
+import { siteConfig } from "@/config/site";
+import { cn } from "@/lib/utils";
+import { Icons } from "@/components/icons";
+
+const stars = 1000;
+
+export default function Home() {
+  return (
+    <>
+      <section className="container">
+        <div className="flex flex-col items-center justify-between gap-8">
```

Comment:
Since the aim is to keep all the text in the landing page to lower case, I think its good to add a tailwind class: 
`lowercase` here